### DISCUSSION TITLE
SB Lern - Release 5.2 & 5.3 #51

### DISCUSSION THREAD

**Discussion by Shakthieshwari** [Sunbird-Lern/Community#52] [Query]
@rhwarrier Can you please let us know the release plan of Sunbird Lern Building Block for 5.2 and 5.3 release ?

Cc- @aishwaryashikshalokam @Prateek-slokam @aks30 @vijiurs

**Comment by rhwarrier** [comment]
This will be shared for review and inputs as a new thread on the Lern Discussion forum in a week or so.

**Reply by Shakthieshwari** [reply]
Thanks @rhwarrier . Do Please keep us posted on this...

**Reply by Shakthieshwari** [reply]
@rhwarrier Any update on this please?

**Comment by rhwarrier** [comment]
This is work in progress, and should be published by the end of this week - if there are inputs your team has for the BB release, please start a new thread on the forum to discuss the same.

**Reply by Shakthieshwari** [reply]
Sure @rhwarrier

**Comment by rhwarrier** [comment]
Please refer to : https://github.com/Sunbird-Lern/Community/discussions/59

**Comment by Namratakushwah** [comment]
Hello All,

Please find the Sunbird Lern Building block 5.3 sign-off here.

Environment: [[[Dev lern](https://dev.lern.sunbird.org/)]
Testing Status: Completed
Defects Dashboard: [[Link](https://project-sunbird.atlassian.net/browse/LR-578)]

Execution Sheet: [[Link](https://docs.google.com/spreadsheets/d/10_DhYT6UTpq9ZwaKwnIc1v57qf40sjUCKGM-U6MzJtk/edit#gid=0)]
Scope : [[Link](https://project-sunbird.atlassian.net/issues/?filter=12786&jql=project%20%3D%20LR%20AND%20issuetype%20in%20(Bug%2C%20Documentation-Issue%2C%20Installation-Issues%2C%20Minor-Enhancement%2C%20RFC)%20AND%20Sprint%20in%20(353%2C%20354)%20ORDER%20BY%20assignee%20ASC%2C%20cf%5B10010%5D%20ASC)]
Building block sign-off template: [[Link](https://docs.google.com/spreadsheets/d/1-q_lBnrRVVJXCh2Pz4FYL5E3ekSZB5nTLfEM0nZISnY/edit#gid=0)]

cc: @rhwarrier @reshmi-nair @chitranshu-keshav @ashokreddy1208 @PoornimaRajesh @vinukumar-vs @Krishnaj20
